app/src/main/AndroidManifest.xml

```diff
@@ -33,6 +34,11 @@
                 android:resource="@xml/file_paths" />
         </provider>
 
+        <provider
+            android:name="androidx.startup.InitializationProvider"
+            android:authorities="${applicationId}.androidx-startup"
+            tools:node="remove" />
+
```

Comment:
I think you do not need Startup Library here (actually, I did-not work with it, but it seems that for now you do not need it)